build.gradle

```diff
@@ -3,13 +3,15 @@
 buildscript {
     repositories {
         jcenter()
+//        google()
         maven {
             url 'https://maven.google.com/'
             name 'Google'
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.2'
+        classpath "com.android.tools.build:gradle:3.1.1"
```

Comment:
retrieve back the 3.1.2 v

Reply:
can you check that v 3.1.2 works for you? It did not work for me and for Vitali